node/resolvers/document/index.ts

```diff
@@ -21,6 +22,30 @@ export const queries = {
     )(data)
   },
 
+  documentsWithPaging: async (_: any, args: DocumentsArgs, context: Context) => {
+    const { acronym, fields, page, pageSize, where, schema } = args
+    const { clients: { masterdata } } = context
+    const fieldsWithId = union(fields, ['id'])
+    const [documentsData, pageInfo] = await all([
+      masterdata.searchDocuments(acronym, fieldsWithId, where, {
+        page,
+        pageSize,
+      }, schema) as any,
+      masterdata.searchDocumentRaw(acronym, fieldsWithId, where, {
```

Comment:
Why are you doing the same query twice?

Just do one, the raw one, and use the info and data returned by it.

Reply:
Thank you for the guidance. Fixed